Bottom: 69ac75bd63ef617622dd74a5e4e5c6fdfd15a0e4
Top:    c152cfa9cb159d0d15765d402abf6e2c05316835
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-07 15:34:53 +0100

doc: Don't make any DVI targets

We don't have rules to build all the prerequisites, and we don't care
about DVI. This makes "make distcheck" pass again.



---

diff --git a/doc/Makefile.am b/doc/Makefile.am
index 6d1e457..55f8fed 100644
--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -49,6 +49,8 @@ dist_man_MANS=	pcb.1
 html_docs=	${info_TEXINFOS:.texi=.html}
 pdf_docs=	${info_TEXINFOS:.texi=.pdf} refcard.pdf
 
+DVIS=
+
 # put the html manual into 1 file instead of multiple files?
 AM_MAKEINFOHTMLFLAGS=	--css-include=$(srcdir)/pcb.css --no-split
